README.md

```diff
@@ -20,46 +20,46 @@ It is designed to track and locate Narcotics Anonymous meetings, which are regul
 
 The original intended clientele is Narcotics Anonymous Service bodies (although other 12 step fellowships have started
 using BMLT as well). The service body implements a BMLT server, and provides the server to other NA Service bodies.
-This project is the "root" server for the BMLT. It is the "server" part of the BMLT "client/server" architecture.
+This project is the server for the BMLT. It is the "server" part of the BMLT "client/server" architecture.
 
 You can find out more about the BMLT on the [website](https://bmlt.app).
 
-[Follow this link to access the BMLT Root Server GitHub repository](https://github.com/bmlt-enabled/BMLT-Root-Server).
+[Follow this link to access the BMLT Server GitHub repository](https://github.com/bmlt-enabled/BMLT-server).
 There are also links to various predecessor legacy repositories [here](#older-repositories) at the end of this README.
-For specific information on setting up a development environment for work on the BMLT root server, please
+For specific information on setting up a development environment for work on the BMLT server, please
 see [CONTRIBUTING.md](CONTRIBUTING.md) in the GitHub repository.
 
 REQUIREMENTS
 ------------
 
-The root server requires a MySQL 5.7 database or higher and PHP 8.2 or higher.In addition, the following PHP modules
+The server requires a MySQL 5.7 database or higher and PHP 8.2 or higher.In addition, the following PHP modules
 are required: `curl gd intl mbstring mysql xml zip`. In particular, make sure you have `intl` -- this one is more
 likely to be missing. There are a variety of MySQL modules for PHP, and others will probably work as a substitute
 for `mysql` itself.
 
 For more information about server requirements, see the "Things You Will Need Before You Install" section of
-[Installing a New Root Server](https://bmlt.app/setting-up-the-bmlt/).
+[Installing a New Server](https://bmlt.app/setting-up-the-bmlt/).
  
 INSTALLATION
 ------------
 
-For instructions on installing the root server, see [installation/README.md](installation/README.md) in the GitHub repository.
+For instructions on installing the server, see [installation/README.md](installation/README.md) in the GitHub repository.
 
 OLDER REPOSITORIES
 ------------------
 
 The first BMLT release was in 2009. 
 
 Here are various legacy repositories that are predecessors to the
-[current BMLT Root Server GitHub repository](https://github.com/bmlt-enabled/BMLT-Root-Server).
+[current BMLT Server GitHub repository](https://github.com/bmlt-enabled/BMLT-server).
```

Comment:
Probably need to undo the changes to the legacy URLs (or just delete this part -- not sure we still need the links to the really old repos)